<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedEnvelopeDlg</class>
 <widget class="QDialog" name="RedEnvelopeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RedEnvelopeDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_title{
background:#2f2f2f;
}
#btn_confirm{
background:transparent;
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#btn_confirm:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#btn_confirm:pressed{
margin:1px;
padding:1px;
}
#btn_cancel{
background:transparent;
background-image: url(:/TQLive/images/btn_cancel.png);
background-repeat:no-repeat;
border:none;
}
#btn_cancel:hover{
background-image: url(:/TQLive/images/btn_cancel_hover.png);
}
#btn_cancel:pressed{
margin:1px;
padding:1px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:transparent;
}
QLineEdit{
border:1px solid #585858;
border-radius:4px;
background:transparent;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#777777;
padding-left:10px;
}
#btn_recharge{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#4AB134;
background:#3F3F3F;
border-radius:2px;
padding-left:0px;
padding-right:0px;
}
#btn_recharge:pressed,#btn_recharge:hover{
margin:1px;
padding:1px;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}
#btn_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#btn_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#btn_close:pressed{
margin:1px;
padding:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="text">
            <string>发送随机红包</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>39</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_current_goldVal">
          <property name="text">
           <string>当前余额：0探球币</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_recharge">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>去充值</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>26</number>
        </property>
        <property name="leftMargin">
         <number>39</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_amount">
          <property name="text">
           <string>金额：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_amount">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>红包金额</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>26</number>
        </property>
        <property name="leftMargin">
         <number>39</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_number">
          <property name="text">
           <string>个数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_number">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>不得大于红包金额</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>26</number>
        </property>
        <property name="leftMargin">
         <number>39</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_title_">
          <property name="text">
           <string>标题：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_title">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>红包标题，限10字以内</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>86</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_confirm">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
